aioauth/errors.py

```diff
@@ -85,6 +85,21 @@ class InvalidClientError(OAuth2Error[TRequest]):
     error: ErrorType = "invalid_client"
     status_code: HTTPStatus = HTTPStatus.UNAUTHORIZED
 
+    def __init__(
+        self,
+        request: TRequest,
+        description: Optional[str] = None,
+        headers: Optional[HTTPHeaderDict] = None,
+    ):
+        super().__init__(request, description, headers or HTTPHeaderDict())
+
+        auth_values = ["error={}".format(self.error)]
```

Comment:
You will find that this is pretty similar to what [oauthlib](https://github.com/oauthlib/oauthlib/blob/7d7fe904af504c20f7d802650e54a78e3d0be8ba/oauthlib/oauth2/rfc6749/errors.py#L106-L111) and [authlib](https://github.com/lepture/authlib/blob/70a6bfa92aa1007515465a7af1a60a4518abe699/authlib/oauth2/rfc6749/errors.py#L91-L100) do.

Reply:
nice catch!

nitpick: can we use f string instead of `.format` method?

```suggestion
        auth_values = [f"error={self.error}"]
```